src/librbd/io/ImageRequest.h

```diff
@@ -99,6 +100,14 @@ class ImageRequest {
     return m_trace;
   }
 
+  bool needs_throttle() {
+    return throttle_flag;
+  }
+
+  void update_needs_throttle(bool need) {
```

Comment:
Nit: rename to ```set_throttled()``` -- no need for boolean argument since it will only ever be set

Reply:
ok